api/proto/auth/config.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2015-2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package auth
+
+import (
+	"crypto/tls"
+	"time"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/trace"
+)
+
+// Config contains configuration of the client
+type Config struct {
+	// AuthAddrs is a list of addresses to dial
+	AuthAddrs []string
```

Comment:
Maybe call this just `Addrs` because it can point to the proxy or auth server. Maybe add a comment explaining how and why that works (reverse tunnnel or IoT mode).